pkg/datapath/linux/bigtcp/bigtcp.go

```diff
@@ -219,31 +219,31 @@ func InitBIGTCP(bigTCPConfig *Configuration) {
 		// modification and end up with an error, so best to be conservative
 		// and always reset all on error
 		modifiedDevices = append(modifiedDevices, device)
-		if haveIPv4 {
-			err = setGROGSOIPv4MaxSize(device,
-				bigTCPConfig.groIPv4MaxSize, bigTCPConfig.gsoIPv4MaxSize)
+		if haveIPv6 {
```

Comment:
nit: should we leave an comment for this tricky order directly in code? Or hide such logic in setGROGSOIPv6MaxSize to make sure IPv6 operation will not impact IPv4 values.

Reply:
This will make 'setGROGSOIPv6MaxSize' not so clean, by my understanding. And this logic is in kernel, git commit message should be fine, no bugs, then peope will be fine about this. ; - )

Reply:
After thinking more, looks like make sense to add some comments. Thanks!

Reply:
I will re-submit with new logs after tweaking the MAX size.

And yes, if you don't start to code it. ;-) It should be about tweaking internal bigTCP{GRO/GSO}MaxSize, right ?

Reply:
@haiyuewa if you could help with testing on ice, I could code this up.

Reply:
@borkmann Looks like more things need to be considered. Sure, I can help on test it.

Reply:
> Looks like more things need to be considered.

Could you elaborate?

Reply:
Btw, I just checked the veth side, and here we should be good as v6 is adjusted before v4: https://github.com/cilium/cilium/pull/26172/commits/0ab201b39b55ab36e6bd6ba98d60f5da851afe73